Gemfile

```diff
@@ -99,7 +100,7 @@ group :development do
 end
 
 group :test do
-  gem 'capybara'
+  gem 'capybara', '3.19.0'
```

Comment:
we don't want 3.19.1?

Reply:
3.19.0 is what we have been testing with.   Capybara is up to 3.30, or something like that.   I think that we should wait until we get all of the other Gem upgrades & removals completed then we should test with the latest version capybara.